src/lib/ai/mental-arena/MentalArenaFactory.ts

```
@@ -1,101 +1,205 @@
-      (varName) => !process.env[varName],
-    )
-    if (missingVars.length > 0) {
-      throw new Error(
-        `Missing required environment variables: ${missingVars.join(', ')}`,
-      )
+  private static pythonBridgeInstance: MentalArenaPythonBridge | null = null
+
+  static async getPythonBridge(
+    config?: PythonBridgeConfig,
+  ): Promise<MentalArenaPythonBridge> {
+    if (!this.pythonBridgeInstance && config) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)